docs/dev/new_check_howto.md

```diff
@@ -269,58 +269,57 @@ The check is almost done. Let's add the final touches by adding the integration
 
 The set of assets created by the ddev scaffolding must be populated in order for a check to be considered for inclusion:
 
-- **`README.md`**: This contains the documentation for your Check, how to set it up, which data it collects, etc..
-- **`conf.yaml`**: This contains all configuration options for your Agent check. [See the configuration file reference documentation to learn its logic.][10]
-- **`manifest.json`**: This contains the metadata for your Agent Check like its title, its categories... [See the manifest reference documentation to lean more.][11]
+- **`README.md`**: This contains the documentation for your Check, how to set it up, which data it collects, etc…
+- **`spec.yaml`**: This is used to generate `conf.yaml.example` using the `ddev` tooling (see the "Configuration template" tab below). [See the configuration specification documentation to learn more.][16]
+- **`conf.yaml.example`**: This contains default (or example) configuration options for your Agent Check. Do not edit this file by hand—it is generated from the contents of `spec.yaml`. [See the configuration file reference documentation to learn its logic.][10]
```

Comment:
```suggestion
- **`conf.yaml.example`**: This contains default (or example) configuration options for your Agent Check. Do not edit this file by hand as it is generated from the contents of `spec.yaml`. [See the configuration file reference documentation to learn its logic.][10]
```

Reply:
The use of emdash here is correct, but "as" is _also_ correct, so sure. ;)

Reply:
Up to you!